core/pubnub_revoke_token.h

```diff
@@ -0,0 +1,4 @@
+/* -*- c-file-style:"stroustrup"; indent-tabs-mode: nil -*- */
+enum pubnub_res pubnub_revoke_token(pubnub_t* pb, char const* token);
+
+pubnub_chamebl_t pubnub_get_revoke_token_response(pubnub_t* pb);
```

Comment:
@650elx functions defined as part of public interface and some docs maybe useful here.

Reply:
👌